service/raft/group0_state_machine.cc

```diff
@@ -377,6 +377,49 @@ future<> group0_state_machine::merge_and_apply(group0_state_machine_merger& merg
     co_await _sp.mutate_locally({std::move(history)}, nullptr);
 }
 
+static void ensure_group0_schema(const group0_command& cmd, const replica::database& db) {
+    auto validate_schema = [&db](const std::vector<canonical_mutation>& mutations) {
+        for (const auto& mut : mutations) {
+            // Get the schema for the column family
+            auto schema = db.find_schema(mut.column_family_id());
+            if (!schema) {
+                on_internal_error(slogger, "ensure_group0_schema: schema not found");
+            }
+
+            static const db::config cfg;
+            static const auto keyspace_tables = db::system_keyspace::all_tables(cfg);
```

Comment:
shouldn't it be named `system_tables`?

All tables are "keyspace" tables in a sense

Reply:
this would return all system tables, but removed anyway as there would be another problem (e.g. if someone would add a group0 table to some other keyspace)